spec/lib/spec/lib/user_comm_screener_spec.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe UserCommScreener do
+  fab!(:target_user1) { Fabricate(:user, username: "bobscreen") }
+  fab!(:target_user2) { Fabricate(:user, username: "hughscreen") }
+  fab!(:target_user3) { Fabricate(:user, username: "alicescreen") }
+  fab!(:target_user4) { Fabricate(:user, username: "janescreen") }
+  fab!(:target_user5) { Fabricate(:user, username: "maryscreen") }
+
+  before do
+    MutedUser.create!(user: target_user1, muted_user: acting_user)
+    IgnoredUser.create!(user: target_user2, ignored_user: acting_user, expiring_at: 2.days.from_now)
+    target_user3.user_option.update(allow_private_messages: false)
```

Comment:
I think this can be moved into the fabrication of `target_user3` as well.

Reply:
Yep fixing